app/helpers/state_file/nj_2450_helper.rb

```diff
@@ -9,5 +9,22 @@ def get_persons_w2s(intake, primary_or_spouse)
       ssn = primary_or_spouse == :primary ? intake.primary.ssn : intake.spouse.ssn
       intake.state_file_w2s.all&.select { |w2| w2.employee_ssn == ssn }
     end
+
+    def get_employer_name(w2, truncate: false)
+      truncate ? sanitize_for_xml(w2.employer_name, 35) : w2.employer_name
+    end
+
+    def get_wages(w2)
+      w2.wages&.round
+    end
+
+    def get_column_a(w2)
+      column_a = w2.box14_ui_wf_swf&.positive? ? w2.box14_ui_wf_swf : w2.box14_ui_hc_wd
```

Comment:
[dust] works either way, but this can be `column_a = w2.get_box14_ui_overwrite` since that is used elsewhere and is 

```
def get_box14_ui_overwrite
  box14_ui_wf_swf || box14_ui_hc_wd
end
```